faults.csv

```diff
@@ -1,63 +1,70 @@
+USL,Upstream liquid level out of tolerance Alarm,Cryomodule liquid level out of tolerance,Call on shift cryo operator,CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CLL:CM{CRYOMODULE}:2601:US:,LVL.SEVR,,2,2,,
+USL,Upstream liquid level out of tolerance Warning,Cryomodule liquid level out of tolerance,Call on shift cryo operator,CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CLL:CM{CRYOMODULE}:2601:US:,LVL.SEVR,,1,1,Upstream liquid level out of tolerance,
+USH,Upstream HOM temperature is too high,Upstream HOM temperature is too high (could indicate an impending quench),Turn off RF,CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CTE:CM{CRYOMODULE}:1{CAVITY}18:UH:,TEMP.SEVR,,2,2,Upstream HOM temperature is too high,
+DSH,Downstream HOM temperature too high,Downstream HOM temperature too high (could indicate an impending quench),Turn off RF,CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CTE:CM{CRYOMODULE}:1{CAVITY}20:DH:,TEMP.SEVR,,2,2,Downstream HOM temperature too high,
+AOT,Amplitude out of tolerance,The amplitude readback is too far away from the desired amplitude,Run auto setup,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,SELCHAR=3,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,AACTMEAN.SEVR,0,,2,Amplitude out of tolerance,
+POT,Phase out of tolerance,The phase readback is too far away from the desired phase,Check that this isn't being caused by beam feedbacks,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PACTMEAN.SEVR,0,,2,Phase out of tolerance,
+QBP,Quench Interlock Bypassed,Quench interlock bypassed,Contact the on call SRF person,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,QUENCH_BYP_RBV,,1,1,Quench Interlock Bypassed,
+AFB,Feedback Amplitude out of tolerance,The feedback maintaining the amplitude is unable to keep the amplitude within tolerance. (Must be checked after RFREADYFORBEAM). Amplitude feedback of measured gradient is too low,Run auto setup,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,AMPFB_SUM,0,,1,Feedback Amplitude out of tolerance,
+PFB,Phase feedback out of tolerance,The feedback maintaining the phase is unable to keep the phase within tolerance. (Must be checked after RFREADYFORBEAM),Run auto setup,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PHAFB_SUM,0,,1,Phase feedback out of tolerance,
+MGT,XCM magnet PS tripped,Magnet power supply tripped off,Look at the PS Controller Hardware panel to see what fault tripped off the magnet. CATER and reset,CM,,,,,XCOR:{LINAC}:{CRYOMODULE}85:,STATMSG,,8,2,Magnet PS Tripped,

```

Comment:
Oops, do we want these magnet power supply faults released into production too? I believe they'll currently show up as invalid purple MGT faults for the HLs because apparently the HLs don't have these magnet PVs associated with them. Should we just delete the PV suffix of STATMSG on the fault spreadsheet that way these won't be read in our utils.py file and therefore won't be in the backend?